src/insecure/plaintext.js

```diff
@@ -4,18 +4,23 @@ const debug = require('debug')
 const log = Object.assign(debug('libp2p:plaintext'), {
   error: debug('libp2p:plaintext:err')
 })
+// @ts-ignore it-handshake do not export types
 const handshake = require('it-handshake')
 const lp = require('it-length-prefixed')
 const PeerId = require('peer-id')
 const { UnexpectedPeerError, InvalidCryptoExchangeError } = require('libp2p-interfaces/src/crypto/errors')
 
+// @ts-ignore protons do not export types
 const { Exchange, KeyType } = require('./proto')
 const protocol = '/plaintext/2.0.0'
 
 /**
  * @typedef {import('libp2p-interfaces/src/connection').Connection} Connection
  */
 
+/**
+ * @param {{ id: Uint8Array; pubkey: { Type: any, Data: Uint8Array; }; }} exchange
```

Comment:
potential candidate for a named type?

Reply:
fixed by relying on protobufjs instead of protons, which offers us all the types for protobuf definitions